safe_transaction_service/account_abstraction/serializers.py

```diff
@@ -207,6 +224,42 @@ def validate_valid_until(
     def validate(self, attrs):
         attrs = super().validate(attrs)
 
+        entry_point = attrs["entry_point"]
+        module_address = attrs["module_address"]
+        is_v7 = entry_point.lower() == ENTRYPOINT_V7.lower()
```

Comment:
Again, please don't use entrypoint address and find another alternative like fields even if it's not so "correct"

Reply:
Both useroperation types use the same class, so they can be identical(except for the entrypoint address). I can add some sanity checks.